pkg/kubelet/kubelet.go

```diff
@@ -74,8 +75,10 @@ const (
 	initialNodeStatusUpdateFrequency = 100 * time.Millisecond
 	nodeStatusUpdateFrequencyInc     = 500 * time.Millisecond
 
-	// The retry count for updating node status at each sync period.
-	nodeStatusUpdateRetry = 5
+	// Node status update frequency and retry count. Note: be cautious when changing nodeStatusUpdateFrequency,
+	// it must work with nodecontroller.nodeMonitorGracePeriod.
+	nodeStatusUpdateFrequency = 2 * time.Second
```

Comment:
We deliberately change this to a constant.  It's somewhat internal mechanism, and using a wrong value will result in an unstable state. #5265